src/algebra/algebra/subalgebra.lean

```diff
@@ -960,6 +960,34 @@ lemma smul_mem_pointwise_smul (m : R') (r : A) (S : subalgebra R A) : r ∈ S 
 
 end pointwise
 
+section center
+
+lemma algebra_map_mem_center (r : R) : algebra_map R A r ∈ subsemiring.center A :=
```

Comment:
```suggestion
lemma _root_.set.algebra_map_mem_center (r : R) : algebra_map R A r ∈ set.center A :=
```

Reply:
Aha, now I understand. I did note I had dropped your `set` prefix but I had missed the point about `set.center`. Thanks; this also answers a lingering concern I had about the `subring` version.